pony.g

```diff
@@ -209,7 +209,7 @@ tuple
   ;
 
 lambdacaptures
-  : ('(' | LPAREN_NEW) lambdacapture (',' lambdacapture)* ')'
+  : ('(' | LPAREN_NEW) (lambdacapture | 'this') (',' (lambdacapture | 'this'))* ')'
```

Comment:
Hm, I looked a bit and couldn't find such a divergence, so I guess I might be wrong.